test/tests/functional/pbs_multi_sched.py

```diff
@@ -796,24 +796,24 @@ def test_fairshare_usage(self):
         self.server.expect(JOB, {'job_state': 'R'}, id=sc1_jid1)
         self.server.expect(JOB, {'job_state': 'Q'}, id=sc1_jid3)
         self.server.expect(JOB, {'job_state': 'Q'}, id=sc1_jid2)
-        self.server.manager(MGR_CMD_SET, SCHED, {'scheduling': 'True'},
-                            id='sc1')
         # need to delete the running job because PBS has only 1 ncpu and
         # our work is also done with the job.
         # this step will decrease the execution time as well
+        self.server.manager(MGR_CMD_SET, SCHED, {'scheduling': 'True'},
+                            id='sc1')
```

Comment:
What's the purpose of the manager() call?  Each of the submits above caused a cycle.  The delete below will cause one as well.

Reply:
We need to run a scheduling cycle to make sure that the scheduler records fairshare usage data.

Reply:
Got it, maybe add a comment to say that.  Plus, depending what your fairshare_usage_res is, the mom might not have reported any usage and you'll still have 0.

Reply:
the test uses ncpus*100, so we don't use mom values.